README.md

```diff
@@ -110,174 +113,148 @@ server.start((err) => {
   - `args` - array of cmd line arguments to be passed to ipfs daemon
   - `config` - ipfs configuration options
-  - `exec` - path to the desired IPFS executable to spawn, otherwise `ipfsd-ctl` will try to locate the correct one based on the `type`. In the case of `proc` type, exec is required and expects an IPFS coderef
-  
- - `callback` - is a function with the signature `cb(err, ipfsd)` where:
-   - `err` - is the error set if spawning the node is unsuccessful
-   - `ipfsd` - is the daemon controller instance: 
-     - `api` - a property of `ipfsd`, an instance of  [ipfs-api](https://github.com/ipfs/js-ipfs-api) attached to the newly created ipfs node   
-
-### IPFS Daemon Controller (ipfsd)
-
-> The IPFS daemon controller that allows interacting with the spawned IPFS process
 
-#### `apiAddr` (getter) 
+`callback` - is a function with the signature `function (err, ipfsd)` where:
+  - `err` - is the error set if spawning the node is unsuccessful
+  - `ipfsd` - is the daemon controller instance: 
+    - `api` - a property of `ipfsd`, an instance of  [ipfs-api](https://github.com/ipfs/js-ipfs-api) attached to the newly created ipfs node   
 
-> Get the address (multiaddr) of connected IPFS API.
+### IPFS Daemon Controller (`ipfsd`)
 
-- returns multiaddr
+The IPFS daemon controller (`ipfsd`) allows you to interact with the spawned IPFS daemon.
 
-#### `gatewayAddr` (getter)
+#### `ipfsd.apiAddr` (getter)
 
-> Get the address (multiaddr) of connected IPFS HTTP Gateway.
+Get the address (multiaddr) of connected IPFS API. Returns a multiaddr,
 
-- returns multiaddr
+#### `ipfsd.gatewayAddr` (getter)
 
-#### `repoPath` (getter)
+Get the address (multiaddr) of connected IPFS HTTP Gateway. Returns a multiaddr.
 
-> Get the current repo path.
+#### `ipfsd.repoPath` (getter)
 
-- returns string
+Get the current repo path. Returns string
 
-#### `started` (getter)
+#### `ipfsd.started` (getter)
 
->  Is the node started.
-
-- returns boolean
+Is the node started. Returns a boolean.
  
-#### `init ([initOpts], callback)`
+#### `init([initOpts], callback)`
 
-> Initialize a repo.
+Initialize a repo. 
 
-- `initOpts` (optional) - options object with the following entries
+`initOpts` (optional) is an object with the following properties:
   - `keysize` (default 2048) - The bit size of the identiy key.
   - `directory` (default IPFS_PATH if defined, or ~/.ipfs for go-ipfs and ~/.jsipfs for js-ipfs) - The location of the repo.
-  - `function (Error, Node)` callback - receives an instance of this Node on success or an instance of `Error` on failure
-
-
-#### `cleanup (callback)`
-
-> Delete the repo that was being used. If the node was marked as `disposable` this will be called automatically when the process is exited.
+ 
+`callback` is a function with the signature `function (Error, Node)` where `err` is an Error in case something goes wrong and `Node` is a ???.
```

Comment:
It returns an ipfsd controller (`ipfsd`). I'll update this branch with the change.